rskj-core/src/test/java/co/rsk/peg/PowpegMigrationTest.java

```diff
@@ -275,13 +276,54 @@ private void testChangePowpeg(
         /*
           Activation phase
          */
-
         // Move the required blocks ahead for the new powpeg to become active
         // (overriding block number to ensure we don't move beyond the activation phase)
-        blockNumber = initialBlock.getNumber() + bridgeConstants.getFederationActivationAge();
+        blockNumber = initialBlock.getNumber() + bridgeConstants.getFederationActivationAge(activations);
         Block activationBlock = mock(Block.class);
         doReturn(blockNumber).when(activationBlock).getNumber();
 
+        // assuming fed activation age after rskip383 is greater than legacy fed activation age, we can check that new fed
+        // should not be active at the legacy activation age when RSKIP383 is active
+
+        if (activations.isActive(ConsensusRule.RSKIP383)){
+            ActivationConfig.ForBlock activationsBeforeRSKIP383 = mock(ActivationConfig.ForBlock.class);
```

Comment:
Why not simply use hop activations?

Reply:
Because that's the only RSKIP we care for the `getFederationActivationAge` method. I think it's better to keep it specific for assertions.